src/components/v5/common/ActionSidebar/partials/consts.tsx

```diff
@@ -5,11 +5,11 @@ import { ACTION } from '~constants/actions';
 
 import { CardSelectOption } from '../../Fields/CardSelect/types';
 
-export const DISTRIBUTION_METHOD = {
-  Equal: 'equal',
-  Unequal: 'unequal',
-  ReputationPercentage: 'reputation-percentage',
-} as const;
+export enum DISTRIBUTION_METHOD {
+  Equal = 'equal',
+  Unequal = 'unequal',
+  ReputationPercentage = 'reputation-percentage',
+}
 
 export type DistributionMethod =
   (typeof DISTRIBUTION_METHOD)[keyof typeof DISTRIBUTION_METHOD];
```

Comment:
This is wholly on me for not being specific enough in the issue but the idea behind switching to `enum`s was that we can ditch those types. Could you replace those types with newly added enums everywhere?